dashboard/app/controllers/plc/course_units_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Plc::CourseUnitsController < ApplicationController
+  load_and_authorize_resource
+
+  # GET /launch/
+  def launch
+  end
+
+  # POST /launch_plc_course/
+  def launch_plc_course
+    course_unit = Plc::CourseUnit.find(params[:plc_course_unit_id])
+    raise ActiveRecord::RecordNotFound unless course_unit
+    course_unit.launch
+    course_unit.reload
+
+    notice_string = if course_unit.started
+                      "#{course_unit.unit_name} was Launched"
+                    else
+                      "#{course_unit.unit_name} was failed to launch"
```

Comment:
Nit: just "failed to launch" for else case? And consistent capitalization/not in "launch"?